tools/ansible/roles/dockerfile/files/launch_awx.sh

```diff
@@ -17,4 +17,13 @@ set -e
 
 wait-for-migrations
 
-supervisord -c /etc/supervisord.conf
+awx-manage provision_instance
+
+# This file will be re-written when the dispatcher calls reconfigure_rsyslog(),
```

Comment:
Can you paste the error from that?

Reply:
Pretty straight forward to test it out:

```
bash-5.1$ rsyslogd -n -f /var/lib/awx/rsyslog/rsyslog-doesnt-exist-yet.conf
rsyslogd: could not open config file '/var/lib/awx/rsyslog/rsyslog-doesnt-exist.conf': No such file or directory [v8.2102.0-101.el9 try https://www.rsyslog.com/e/2104 ]
rsyslogd: run failed with error -2104 (see rsyslog.h or try https://www.rsyslog.com/e/2104 to learn what that number means)
```

Reply:
@shanemcd if we are now creating a dummy file (that will be overwritten) in the launch script, then we may not need to copy in a dummy file in the Dockerfile anymore - https://github.com/ansible/awx/blob/1de9dddd21b0bd7144137f8a8a2956f4f097bccd/tools/ansible/roles/dockerfile/templates/Dockerfile.j2#L195

Reply:
@rooftopcellist I think that's still used by the development environment... But I can do the same thing in `tools/docker-compose/launch_awx.sh` as I did here.